ci/test_imagefile

```diff
@@ -0,0 +1,10 @@
+ARG CI_IMAGE_NAME_TAG
+FROM ${CI_IMAGE_NAME_TAG}
+
+ARG FILE_ENV
+ENV FILE_ENV=${FILE_ENV}
+
+COPY ./ci/retry/retry /usr/bin/retry
+COPY ./ci/test/00_setup_env.sh ./${FILE_ENV} ./ci/test/01_base_install.sh /ci_base_install/ci/test/
+
+RUN ["bash", "-c", "cd /ci_base_install/ && set -o errexit && source ./ci/test/00_setup_env.sh && ./ci/test/01_base_install.sh"]
```

Comment:
To get even more of a speedup, wouldn't we want to move these steps into `RUN` commands earlier in the Dockerfile, and then just generate a few separate images on the basis of `CI_IMAGE_NAME_TAG`/`FILE_ENV`?

Reply:
The packages are defined in FILE_ENV, so RUN needs to be after FILE_ENV

Reply:
I'm sorry, shouldn't be reading this before the caffeine has hit; I misread `RUN` here for `ENTRYPOINT`.